dlc_developer_config.toml

```diff
@@ -48,7 +48,7 @@ sagemaker_local_tests = false
 # "standard" --> run standard sagemaker remote tests from test/sagemaker_tests
 # "rc" --> run release_candidate_integration tests
 # "efa" --> run efa sagemaker tests
-sagemaker_remote_tests = "off"
```

Comment:
Change it to `efa` in order to run `standard` and `efa` tests

Reply:
Done, changed to efa